internal/cluster/port_range.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cluster
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type PortRange struct {
```

Comment:
BTW, where do we use this?

Reply:
I've taken this one from Azure Discovery PR to implement port ranges when the port is not specified. I didn't implement that, so I'd better remove it.

Reply:
Removed at: https://github.com/hazelcast/hazelcast-go-client/pull/573/commits/67db1002572e04ed4e403ea193defa9d8f3beb10